README.md

```diff
@@ -1,16 +1,40 @@
-# Container Object Storage Interface API
+COSI repos:
+
+|Repo | Images | 
+|----------------|-----------|
+| [Spec](https://github.com/kubernetes-sigs/container-object-storage-interface-spec) | N/A  |
+| [API](https://github.com/kubernetes-sigs/container-object-storage-interface-api) | N/A |
+| [Controller](https://github.com/kubernetes-sigs/container-object-storage-interface-controller) | [cosi-controller](https://quay.io/repository/containerobjectstorage/objectstorage-controller?tab=tags) |
+| [Provisioner Sidecar](https://github.com/kubernetes-sigs/container-object-storage-interface-provisioner-sidecar) | cosi-provisioner TODO |
+| [CSI Adapter](https://github.com/kubernetes-sigs/container-object-storage-interface-csi-adapter) | cosi-node-adapter TODO |
+\
```

Comment:
Can you please fix this? And also squash all the commits. Other than that it looks good to me.

Reply:
I asked you about this in https://github.com/kubernetes-sigs/container-object-storage-interface-spec/pull/15 , will git rebase first thing tomorrow.